arrow-core-data/src/main/kotlin/arrow/core/Nullable.kt

```diff
@@ -1,9 +1,6 @@
 @file:Suppress("NAME_SHADOWING")
 package arrow.core
 
-inline fun <A, R> mapN(a: A?, fn: (A) -> R): R? =
```

Comment:
Why these changes unrelated to PR?

Reply:
https://github.com/arrow-kt/arrow-core/pull/189#pullrequestreview-452433217